jax/_src/numpy/lax_numpy.py

```diff
@@ -82,6 +82,7 @@
 from jax.experimental.array import Array
 
 newaxis = None
+NoneType = type(None)
```

Comment:
What's this for?

Reply:
Oops, stale! (I was doing `isinstance(idx.step, NoneType)` etc., but changed when mypy complained.)